CONTRIBUTING.md

```diff
@@ -7,12 +7,13 @@ Thanks for your interest in this project.
 2. [Developer resources](#developer-resources)
 3. [Eclipse Development Process](#eclipse-development-process)
 4. [Eclipse Contributor Agreement](#eclipse-contributor-agreement)
-5. [General contribution to the project](#general-contribution-to-the-project)
-6. [Contributing as a Consultant](#contributing-as-a-consultant)
-7. [Contributing as a Developer](#contributing-as-a-developer)
+5. [Code of Conduct](#code-of-conduct)
+6. [General contribution to the project](#general-contribution-to-the-project)
+7. [Contributing as a Consultant](#contributing-as-a-consultant)
+8. [Contributing as a Developer](#contributing-as-a-developer)
 9. [Contact](#contact)
 
-## Project description
+## Project Description
```

Comment:
Why was the option in capital letters used here? Project description might be correct.